src/pages/iou/MoneyRequestDescriptionPage.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import TextInput from '../../components/TextInput';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Form from '../../components/Form';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Navigation from '../../libs/Navigation/Navigation';
+import compose from '../../libs/compose';
+import * as IOU from '../../libs/actions/IOU';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** Onyx Props */
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        comment: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    iou: {
+        comment: '',
+    },
+};
+
+class MoneyRequestDescriptionPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.updateComment = this.updateComment.bind(this);
+    }
+
+    /**
+     * Sets the IOU comment by saving it to Onyx.
```

Comment:
Probably best to go to as little mentions of IOU as possible
```suggestion
     * Sets the money request comment by saving it to Onyx.
```